src/js/services/optimizely.js

```diff
@@ -2,9 +2,11 @@ import * as optimizelySDK from '@optimizely/optimizely-sdk';
 import { v4 as uuidv4 } from 'uuid';
 import { isProduction } from '../utils';
 import Cookies from 'js-cookie';
+// testing not working unless add this
+import { forceAll } from '../experiments/forceAll';
```

Comment:
?

Reply:
+1, I am not sure I follow

Reply:
I can examplain why I did it but I do not know a better way to fix the issue. I get `ReferenceError: forceAll is not defined`. I think because its a unit test not an E2E test it doesn't know what forceAll is, so if I import forceAll then it knows what it is.

Reply:
why can't you use globs to solve that?

Reply:
glob worked thank you!